frontend/components/doctor/patient-charts-overview.tsx

```diff
@@ -1,12 +1,27 @@
-import { Box, Center, Flex, SimpleGrid, Tab, TabList, TabPanel, TabPanels, Tabs } from "@chakra-ui/react";
+import { Box, Center, SimpleGrid, Tab, TabList, TabPanel, TabPanels, Tabs, Text } from "@chakra-ui/react";
 import { extractStatuses, transformWeightTempData } from "@frontend/functions/data-transform-chart";
 import { Patient } from "@frontend/models/patient";
 import { DAY } from "@frontend/utils/constants";
 import PieChart from "@frontend/components/pie-chart";
 import Table from "@frontend/components/doctor/patient-overview-table";
 import ScatterChart from "@frontend/components/scatter-chart";
+import { CSSProperties, useState } from "react";
 
 export default function PatientChartsOverview({ patientList }: { patientList: Patient[] }) {
+    const today = new Date().toISOString().slice(0, 10);
```

Comment:
I see we are importing/using the DAY constant, so is this still needed?

Reply:
Yep `today` is in the date form (e.g., `2022-03-03`) while DAY constant contains strings like `"today"`, `"yesterday"`, etc